authentik/stages/authenticator_email/templates/email/base.html

```diff
@@ -0,0 +1,131 @@
+{% load authentik_stages_email %}
```

Comment:
this should also not be required to be duplicated here, we can refer to the base email template in the email stage